.eslintrc

```diff
@@ -71,5 +71,8 @@
   "env": {
     "node": true,
     "browser": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018
```

Comment:
Sorry, this only applies to the first (await) impl.  If you prefer the Promise approach, I'll remove this. Just depends on which direction you want to go.